### PR TITLE
Update Brazzers.yml

### PR DESCRIPTION
**Pull request by fustios**
New Site layout. I used the RealityKings.yml as a template, they are quite similar.

### CODE REVIEW

File: scrapers/Brazzers.yml [hunk 1]
```diff
@@ -1,29 +1,21 @@
...
       Performers:
-        Name: $sceneinfo/p[@class="related-model"]/a
-      Image:
-        selector: //div[@id="trailer-player"]/img/@src
-        replace:
-          - regex: ^
-            with: "https:"
+        Name: //div[@class="sc-1b6bgon-2 jywgHS"]/span/a/text()
+      Studio:
+        Name: //div[@class="vdkjux-5 gIliaY"]/text()
```
Comment (bnkai) [inline scrapers/Brazzers.yml:20]: This should be 
```yaml
     Studio:
        Name: //a[@class="vdkjux-5 gIliaY"]/text()
```
instead

**Comment by bnkai** [comment]
Can you also add a comment like this at the end ?
```yaml
# Last Updated August 4, 2020
```

File: scrapers/Brazzers.yml (hunk 1 above)
Comment (fustios) [reply scrapers/Brazzers.yml]: Ah yes, i changed it.

**Comment by fustios** [comment]
Added the update info at the end of the file and changed the Studio path.

**Pull request merged**